### ISSUE TITLE
Fix division of German Vinaya files

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#2557]
It seems there are major issues with the structure of the German Vinaya translation.

https://discourse.suttacentral.net/t/discrepancies-with-chapter-titles-in-vinaya-texts/27162/11

It should be revised so that it aligns as well as possible with the Pali.

Requirements:

- German speaker
- can do HTML
- some familiarity with the Vinaya

**Comment by sabbamitta** [comment]
I believe I have fixed Kd 1, including the link issue https://github.com/suttacentral/suttacentral/issues/2560.

Please merge my pull request https://github.com/suttacentral/sc-data/pull/195, thank you!

**Comment by thesunshade** [comment]
@sabbamitta, I wonder if you can think of any German speaking monastics who might have the skill or interest to continue this work. It seems valuable according to your D&D post. Thanks for making that first fix.

**Comment by sabbamitta** [comment]
I can't think of any at the moment. I was thinking of looking into the rest of the files myself at some point, but it drifted a bit away from my radar (and there are enough of other things to look into, of course …).

Thank you for taking the big broom and sweeping through all these old issues! I'll see what I can do for this one, or if there is anybody else who might want to help out.

**Comment by thesunshade** [comment]
Sorry to have swept you up in my big sweep! Please don't feel any pressure to work on any of these. I was just looking through old issues to add tags and close things that were either done or clearly not going to be implemented.

**Comment by sabbamitta** [comment]
Yes, that's very good! I am just taking a few notes for myself and take them on one by one as I find the time. No worries!

**Comment by sabbamitta** [comment]
Hello Bhante @thesunshade , I just checked this issue, and it seems I have fixed them all about a year or so ago. 

I've done Kd1-10, the ones translated by Maitrimurti/Trätow. The other ones don't have any IDs nor links nor nothing, and they are only excerpts. Apart from the Khandhakas we only have a Bhikkhu and a Bhikkhuni Patimokkha, also with no IDs or section links or anything. So what can be done has been done. :white_check_mark: 

I would close the issue, but it seems I don't have the powers. I don't see a "close" button.
